pom.xml

```
@@ -31,13 +31,20 @@
         <munit.input.directory>src/test/munit</munit.input.directory>
         <munit.citizen.input.directory>src/test/munit-citizen</munit.citizen.input.directory>
         <munit.output.directory>${basedir}/target/test-mule/munit</munit.output.directory>
-        <munit.extensions.maven.plugin.version>1.2.0-SNAPSHOT</munit.extensions.maven.plugin.version>
-        <munit.version>3.1.0-SNAPSHOT</munit.version>
+        <munit.extensions.maven.plugin.version>1.3.0-rc2</munit.extensions.maven.plugin.version>
```

Ale, I pasted the wrong version, my apologies..
```suggestion
        <munit.extensions.maven.plugin.version>1.3.0</munit.extensions.maven.plugin.version>
```